bin/rose-make-docs

```diff
@@ -65,6 +65,30 @@
 #     `linkcheck`
 #         Checks external links.
 #
+# SOFTWARE ENVIRONMENT
+#     The software dependencies can be found by running the command::
+#
+#        $ rose check-software --docs
+#
+#     Rose provides two ways of installing the Python dependencies for the Rose
+#     documentation builder:
+#
+#     Virtual Environment:
+#         Rose will automatically build a Python virtual environment
+#         when the --venv flag is used with this command. The virtual
+#         environment will be created in rose/venv and will be destroyed
+#         after use. Use the --dev flag to prevent the environment being
+#         destroyed for future use. Example::
+#
+#            $ rose make-docs --venv --dev html  # create and keep a virtualenv
+#
+#     Conda
+#         An environment file for creating a conda env can be found in
+#         etc/rose-docs-env.yaml. Example usage::
```

Comment:
Isn't .yml the (standard) extension for YAML files?

Reply:
Both `.yml` and `.yaml` are valid.

*Historical sidenote, quite a lot of formats are like that e.g. JPEG (`.jpg`, `.jpeg`), HTML (`.htm`, `.html`) the reason being that once upon a time not all systems permitted file extensions of more than three characters.*

Reply:
DOS!